src/components/ArticlesPage/ArticlesPage.jsx

```diff
@@ -0,0 +1,32 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import ArticleCard from "./ArticleCard";
+
+function ArticlesPage() {
+    const [articles, setArticles] = useState([]);
+
+    function getAllArticles() {
+        return axios
+            .get("https://northcoders-news-c5lb.onrender.com/api/articles")
+            .then((response) => {
+                return response.data.articles;
+            });
+    };
```

Comment:
This function should be stored away in an api.js file instead of being just out here in the component